app.json

```
@@ -11,14 +11,14 @@
       "description": "Development related config, you can leave this be please.",
       "value": "false"
     },
+    "SDK_VERSION": {
+      "description": "Specify a certain version for the SDK.",
+      "value": "v0"
```

```suggestion
      "value": "v0",
      "required": true
```

So we can avoid any possible case of it being passed empty